worldgenedit.lua

```diff
@@ -0,0 +1,165 @@
+--Edit certain worldgen parameters while it is running.
+--[====[
+
+worldgenedit
+============
+Perhaps of dubious utility in general, being able to alter an active worldgen is handy.
+
+]====]
+local gui = require 'gui'
+local widgets = require 'gui.widgets'
+local wv = dfhack.gui.getCurViewscreen()
+local wd = df.global.world.world_data
+local wg = df.global.world.worldgen.worldgen_parms
+local sites = df.global.world.world_data.sites
+local pent, dent, eent, hent, gent, stn, stc, pop, eny
```

Comment:
A lot of these are hard to understand. Longer, more descriptive variable names would be better. I'd recommend some plugin (or editor) that can autocomplete names for you, if you don't have that ability already, so that typing them out is less annoying.